ios/application_swiftui/control

```diff
@@ -0,0 +1,9 @@
+Package: @@PACKAGENAME@@
+Name: @@FULLPROJECTNAME@@
+Version: 0.0.1
+Architecture: iphoneos-arm
+Description: An awesome SwiftUI application!
+Maintainer: @@USER@@
+Author: @@USER@@
+Section: Utilities
+Depends: ${LIBSWIFT}, firmware (>= 14.0)
```

Comment:
I don't believe the libswift dep is required for newer iOS versions. I installed a test app built with this template w/o libswift installed and the project works as expected.

Reply:
Correct. Given that there's already a requirement for iOS 14, the swift libs are not needed (the libs shipped with iOS starting with iOS 12.3 I believe)

```suggestion
Depends: firmware (>= 14.0)
```

Reply:
In that case, we should also modify the application_swift template's depends (not necessarily in this pr).